kentik_api_library/kentik_api/public/device.py

```diff
@@ -119,6 +122,17 @@ def snmp_speed(self) -> float:
     def initial_snmp_speed(self) -> Optional[float]:
         return self._initial_snmp_speed
 
+    @property
+    def name(self):
+        return self._interface_description
+
+    @property
+    def speed(self):
```

Comment:
I'm not quite sure what you mean with the comment. I agree that there should be more in-line documentation and I can surely add a docstring to the this property. Is that what you've meant?

Reply:
Yes, I just meant adding the docstring.

And I think that we don't need docstrings for simple property functions, such as `initial_snmp_speed(self)`, because it just returns a value of a field documented here: https://kb.kentik.com/v0/Ec05.htm#Ec05-Interface_Object

BTW, Kentik docs state that, SNMP speeds are expressed in Mbps. Is `speed * 10e6 * 8` calculation below needed?

Reply:
Will add docstring. It is going to be a little bit alone, but it will be definitely useful.

Good catch on the unit conversion. It was all wrong! It is funny that nobody noticed until now (and I've shown the analysis results to quite a few people ;-)).

Reply:
@mmac-m3a The question is still valid.

Reply:
Done